content/projects/esg/main.md

```diff
@@ -3,8 +3,6 @@ title: EuroScienceGateway
 description: Improving computational workflows in Galaxy by maturing Pulsar Network and publishing computational workflows as FAIR Digital Objects.
 ---
 
-# EuroScienceGateway Project
```

Comment:
Don't remove the header!

Reply:
The idea was to remove the heading so ce the big logo is underneath with the same text... Should we keep the header ?

Reply:
Stian if you think we should bring it back, let us know and we will do so.